libsession/src/main/java/org/session/libsession/utilities/IdUtil.kt

```diff
@@ -1,4 +1,4 @@
 package org.session.libsession.utilities
 
 fun truncateIdForDisplay(id: String): String =
-    id.takeIf { it.length > 8 }?.apply{ "${take(4)}…${takeLast(4)}" } ?: id
+    id.takeIf { it.length > 8 }?.run{ "${take(4)}…${takeLast(4)}" } ?: id
```

Comment:
lol what an "oops" moment

Reply:
Same fix with tests is here: https://github.com/oxen-io/session-android/pull/1227

Internally tracked as: https://optf.atlassian.net/browse/SES-538